### PR TITLE
pipewire: update to 0.3.65

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by cinerea0** [comment]
This PR was only able to be submitted without significant patching because the devs released a new version of pipewire-media-session compatible with this version of pipewire. It took them a week to do so, and their [release notes](https://gitlab.freedesktop.org/pipewire/media-session/-/releases/0.4.2) indicate that they may not do so again. As such I would like to draw attention to this PR that splits p-m-s out: https://github.com/void-linux/void-packages/pull/38521.

**Comment by oreo639** [comment]
There are some patches listed here:
https://gitlab.freedesktop.org/pipewire/pipewire/-/releases

**Comment by cinerea0** [comment]
Temporarily marking as draft as I test other, non-p-m-s patches linked in the release notes.

**Comment by ahesford** [comment]
For reasons I outlined in https://github.com/void-linux/void-packages/pull/38521#issuecomment-1403743362, I think we should decide how we want to handle p-m-s after the 0.3.64 release. Based on that decision, we should use this bump to ensure continuity while also providing an `INSTALL.msg` to warn users of anything that might break.

**Comment by cinerea0** [comment]
I've added an `INSTALL.msg` file that explains the situation. I worry that it may be too wordy, which is bad because it seems like no one reads them as it is.

File: srcpkgs/pipewire/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Support for pipewire-media-session has been dropped by the pipewire developers,
+and will no longer be included in Void packages of pipewire after 0.3.64. 
+Wireplumber, an alternate session manager, is supported and packaged. If you 
+are currently using pipewire-media-session, you must install wireplumber and 
+change your configuration to use it before the next pipewire release.
```
Comment (ahesford) [inline srcpkgs/pipewire/INSTALL.msg:5]: The pipewire-media-session session manager has been abandoned upstream and will be removed from Void in a subsequent pipewire update. All users are encouraged to transition to wireplumber immediately to avoid unexpected loss of functionality in the future.

Comment (cinerea0) [reply srcpkgs/pipewire/INSTALL.msg]: Changed to this wording.

**Comment by paper42** [comment]
Documentation on how to use wireplumber is missing from the handbook, so before telling people they should use it, we should document it.

**Comment by paper42** [comment]
.65 is released now

**Comment by ahesford** [comment]
https://github.com/void-linux/void-docs/pull/738

@cinerea0 please pull the `wireplumber.desktop` file from #41868 into this PR because I plan to mention it in the docs.

**Comment by cinerea0** [comment]
I've added wireplumber's desktop file. I have to test if 0.3.65 still allows p-m-s to be built. If it does should I change this PR to use 0.3.65 and we shift removing p-m-s to 0.3.66?

**Comment by ahesford** [comment]
If 0.3.65 works with p-m-s, go ahead and bump this. I want to give the INSTALL.msg time to sink in and hopefully get people to transition to wireplumber. We can keep building p-m-s and extend the warning for a few releases if it works without patching.

**Comment by cinerea0** [comment]
p-m-s build correctly locally and CI indicates it builds correctly elsewhere as well.

**Comment by ardadem** [comment]
@ahesford if you plan to use .desktop file for autostart, i want to point out oreo's comment [here](https://github.com/void-linux/void-packages/pull/38521#issuecomment-1399777559) since it makes more sense.

**Comment by ahesford** [comment]
There has been a lot of bikeshedding around pipewire, and I suspect there will continue to be more as we iron out the wrinkles in wireplumber documentation. None of that changes the fact that pipewire-media-session will soon become unmaintainable and we should drop it eventually. People should have the maximum possible notice that this will happen, even if the docs aren't quite ready.
